src/pages/interim/_faq.js

```diff
@@ -5,12 +5,17 @@ import { SectionContainer, Container } from 'components/containers'
 import { localize } from 'components/localization'
 import { Header, Accordion, AccordionItem } from 'components/elements'
 import DotPattern from 'images/svg/dot-pattern.svg'
+import device from 'themes/device'
 
 const AccordionWrapper = styled.div`
     max-width: 79.2rem;
     margin: 0 auto;
     position: relative;
     z-index: 2;
+
+    @media ${device.tablet} {
```

Comment:
it seems this is applied to all accordion, can we put this on the accordion component itself as props or default?

Reply:
This is actually an external wrapper that is wrapping the accordion component. I removed the internal accordion margins for tablet widths and below so that we can better control the gutter spaces left & right for mobile devices through external containers/wrappers.